### DISCUSSION TITLE
How to control volume from esp32 to phone?

### DISCUSSION THREAD

**Discussion by ham0nd** [pschatzmann/ESP32-A2DP#347] [Q&A] [answered]
I want to do this work:
The AVRCP volume adjustment command controls the volume slider on the mobile phone and triggers the volume adjustment window on the mobile phone
but i dont find the code of function from example.
Have the library impleted this function?thx.
@pschatzmann

**Comment by pschatzmann** [comment, marked as answer]
Did you read the related documentation ? https://github.com/pschatzmann/ESP32-A2DP/wiki/Controlling-your-Phone-with-AVRC-Commands

**Comment by Shavo650** [comment]
You can also use Automate app (on Android) to set volume when ESP32 is connected to the phone.
